CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ There are breaking changes in this release:
   - Scripts are now located in `/usr/lib/influxdb/scripts` (previously `/opt/influxdb`)
 
 ### Features
+- [#4841](https://github.com/influxdb/influxdb/pull/4841): Improve parsing of measurements and tags
```

Comment:
Sorry, yes. I wrote that before I made the performance improvements. Not added to the `CHANGELOG` before, how about:

> Improve point parsing speed

Also, I just realised I put it under the `features` section when it started off as a bug fix. Should I move it or leave it where it is?